source/FreeRTOS_Sockets.c

```diff
@@ -1649,6 +1649,9 @@ void * vSocketClose( FreeRTOS_Socket_t * pxSocket )
                     ( ( pxOtherSocket->u.xTCP.bits.bPassQueued != pdFALSE_UNSIGNED ) ||
                       ( pxOtherSocket->u.xTCP.bits.bPassAccept != pdFALSE_UNSIGNED ) ) )
                 {
+                    /* recursive call: the number of rcusion is limited by
+                     * design */
```

Comment:
Change to the following:
~~~
    /* Recursive call: the number of recusions is limited by
     * design. */
~~~